tensor_comprehensions/__init__.py

```diff
@@ -35,6 +35,7 @@
 from tensor_comprehensions.tclib import TcExecutor
 from tensor_comprehensions.tclib import Tuner
 from tensor_comprehensions.tclib import TunerConfig
+from tensor_comprehensions.tclib import shared_memory_size
```

Comment:
This is a quite low-level function, I would keep it in `tclib` and not import by default with `import tensor_comprehensions`.  We really want to keep the high-level API as simple as possible